src/site/content/en/fast/codelab-imgix/index.md

```diff
@@ -0,0 +1,121 @@
+---
+layout: codelab
+title: Using imgix to optimize images
+authors:
+  - sherwinski
+date: 2019-07-31
+description: |
+  In this codelab, learn how to use imgix to optimize and cache your images for a faster website experience.
+glitch: imgix-demo
+related_post: use-imagemin-to-compress-images
+draft: true
+---
+
+[imgix](https://www.imgix.com/) is a service that transforms, optimizes, and caches your images for a faster website experience. When building your web application, it's nearly impossible to account for all the [various devices](https://www.mydevice.io/#compare-devices) and screen sizes that it might be viewed on. For images, this presents a major problem, because they require optimization specific to the context to display quickly and with high quality—both of which are key metrics for engagement and conversion. With imgix, you can ensure that you're always serving images to your users at the ideal size and speed for their specific device. In addition, imgix provides a wide variety of image transformation options, all easily accessible through a robust API of [URL parameters](https://docs.imgix.com/apis/url?utm_source=webdev&utm_medium=referral&utm_campaign=codelab).
```

Comment:
This diff isn't very clear so this is what I changed:
From this:
"For images, this presents a major problem, because they require optimization specific to the context to display quickly and with high quality—both of which are key metrics for engagement and conversion. "
To this:
"For images, this presents a major problem: in order to display images quickly and with high quality (both of which are key metrics for engagement and conversion) they must be optimized based on their context. "